style.css

```diff
@@ -148,10 +148,36 @@ body {
   color: #fff;
   border-radius: 4px;
   background: #0056EC;
+  border: 0;
+  transition: background 0.2s ease-out 0s;
+  outline: none;
 }
 
 .main__button:hover {
   cursor: pointer;
   background: #1e90ff;
-  transition: 1s;
 }
+
+@media (max-width: 1366px;) {
+  .menu__item {
+    font-size: 14px;
+    line-height: 16px;
+  }
+
+  .main__title {
+    font-size: 50px;
+    line-height: 66px;
+  }
+
+  .main__description {
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 21px;
+  }
+
+  .main__button {
+    font-size: 14px;
+    line-height: 16px;
+  }
+}
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/tni-htmlcss2.0/pullRequest?prid=6206132)